etc/bitwarden.profile

```diff
@@ -0,0 +1,54 @@
+# Firejail profile for bitwarden
+# Description: A secure and free password manager for all of your devices
+# This file is overwritten after every install/update.
+# Persistent local customisations
+include bitwarden.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Bitwarden
+ignore noexec /tmp
+noblacklist ${DOWNLOADS}
```

Comment:
Not needed. [disable-xdg.inc](https://github.com/netblue30/firejail/blob/master/etc/disable-xdg.inc) has `blacklist ${DOWNLOADS}` commented out.